src/stdlib.jl

```diff
@@ -1,25 +1,25 @@
 import Base.CoreLogging:
```

Comment:
While here should change to follow style guide?

Reply:
Might want to make an issue as there are probably other cases where this happens throughout the codebase.

Reply:
https://github.com/invenia/Memento.jl/issues/132